helpers/lib/fonts.js

```diff
@@ -87,8 +87,16 @@ module.exports = function(format, themeSettings, handlebars, options) {
     
     const collectedFonts = {};
     _.each(themeSettings, function(value, key) {
-        if (!fontKeyFormat.test(key)) {
+        //check that -font is on end of string but not start of string
```

Comment:
This seems unnecessarily complex. Could we just do the following?

```
const fontKeySuffix = '-font';
if (!key.endsWith(fontKeySuffix)) {
  return;
}
```

It's true that it's slightly different than the current regex, but I think it's safe. If the value doesn't follow the expected format, we will not treat it as a font either.

Reply:
changed.